src/cli/options.ts

```diff
@@ -1,139 +1,182 @@
 import {Options} from "yargs";
-import {Opts, BenchmarkOpts} from "./types.js";
-import {FileCollectionOptions} from "./utils/mochaCliExports.js";
+import {StorageOptions, BenchmarkOpts, FileCollectionOptions} from "../types.js";
 
 export const optionsDefault = {
   threshold: 2,
+  timeoutBench: 10_000,
```

Comment:
Seems like a long timeout.  Do you know what the original value was?

Reply:
Original was based on the mocha configuration. In our repo we had custom value set to `5000`, others users may have different. I believe mocha in general had 2000 as default value, but that seems very low for benchmark operations.